infra.csv

```diff
@@ -38,6 +43,7 @@ Kubernetes,Languages & Frameworks,Adopt,FALSE,"<p>We're using k8s to host a wide
 Terraform,Languages & Frameworks,Adopt,FALSE,"<p>Used to define infrastructure as code, example: <a href=""https://github.com/camunda/infra-core"">infra-core</a> project.</p>"
 Javascript,Languages & Frameworks,Adopt,TRUE,"<p>Used by the <a href=""https://downloads.camunda.cloud"">Download center</a> for Camunda products. <a href=""https://github.com/camunda-internal/camunda-downloads"">repository</a></p>"
 Node.js,Languages & Frameworks,Adopt,TRUE,"<p>Used by the <a href=""https://downloads.camunda.cloud"">Download center</a> for Camunda products. <a href=""https://github.com/camunda-internal/camunda-downloads"">repository</a></p>"
+
```

Comment:
does the newline not break things for the visualizer?

Reply:
Looks like it breaks it, so removed them :+1: